examples/eks-cluster-with-observability/README.md

```diff
@@ -0,0 +1,215 @@
+# EKS Cluster with Amazon Managed Grafana and Amazon OpenSearch Service
+
+This example demonstrates how to use the AWS EKS Accelerator for Terraform to deploy a new Amazon EKS Cluster with Prometheus for metrics and Fluent Bit for logging. It also provisions Amazon Managed Prometheus (AMP), Amazon OpenSearch Service within a VPC, and integrates AMP with Amazon Managed Grafana (AMG). It also deploys an EC2 instance to let us test OpenSearch. Lastly, it includes a sample workload, provisioned with ArgoCD, to generate logs and metrics.
+
+Prometheus sends application metrics to AMP, and AMG ingests data from AMP. Fluent Bit sends logs to Amazon OpenSearch Service.
```

Comment:
Prometheus server collects these metrics and writes to remote Amazon Managed Prometheus endpoint via `remote write` config property. Amazon managed Grafana is used to visulaize the metrics in dashboards by leveraging Amazon Managed Prometheus workspace as a data source. 

AWS FluentBit Addon configured to collects the container logs from EKS Cluster nodes and write to Amazon Open Search service.